modules/tensor_mechanics/src/auxkernels/RankTwoScalarAux.C

```diff
@@ -52,8 +52,34 @@ RankTwoScalarAux::computeValue()
    case 6:
      val = calcEigenValues();
      break;
+   case 7: // VolumetricStrain (Lagrangian)
+     val = _tensor[_qp].trace() +
+           _tensor[_qp].xx()*_tensor[_qp].yy() +
```

Comment:
My suggestion would be to add 
```
enum {x, y, z};
```
here, which would allow you to write 
```
_tensor[_qp](x,x)*_tensor[_qp](y,y)
```
etc. Which has the same number of characters ;-)